main.rb

```diff
@@ -0,0 +1,62 @@
+require './app'
+
+class Start
+  def options
+    [
+      'List all books',
+      'List all people',
+      'Create a person',
+      'Create a book',
+      'Create a rental',
+      'List all rentals for a given person id',
+      'Exit'
+    ]
+  end
+
+  def display_menu
+    puts 'Please choose an option by entering a number:'
+    options.each_with_index { |option, i| puts "#{i + 1} - #{option}" }
+  end
+
+  def call_options
+    {
+      '1' => -> { @app.list_all_books },
+      '2' => -> { @app.list_all_people },
+      '3' => lambda {
+               @app.create_a_person
+               puts "Person created successfully\n"
```

Comment:
- [ ] Kindly fix the error that appears when creating a person. I couldn't create a rental as well because it is dependent on a person and a book.

![person](https://github.com/momo-87/oop-school-library-app/assets/53124548/82c1482b-546a-4158-8de3-c959268cd0a9)